doc/source/modules130-140.rst

```diff
@@ -638,15 +638,15 @@ Module coverage 13.0 -> 14.0
 +--------------------------------------------+-------------------------------------------------+
 |transifex                                   |                                                 |
 +--------------------------------------------+-------------------------------------------------+
-|uom                                         |                                                 |
+|uom                                         | Done                                                |
 +--------------------------------------------+-------------------------------------------------+
-|utm                                         |                                                 |
+|utm                                         | Nothing to do                                                |
```

Comment:
You have to strip extra spaces from the end of the line, but these updates must be done on each PR adding the migration scripts, not apart.